src/CONSTANTS.JS

```diff
@@ -0,0 +1,91 @@
+const letters = {
```

Comment:
Great work filing these in their own constants file! Since we won't be making changes to `letters`, we can make it into a constant `LETTERS`.